download_datasets.py

```
@@ -18,36 +18,68 @@
-    video_learning_events_file_path = f'lang-{language}/video-learning-events.csv'
-    print(f'video_learning_events_file_path: {video_learning_events_file_path}')
-    video_learning_events_dataframe.to_csv(video_learning_events_file_path, index=False)
+    students_csv_url = f'http://{language.lower()}.elimu.ai/analytics/students/students.csv'
+    print(f'students_csv_url: {students_csv_url}')
+    students_dataframe = pandas.read_csv(students_csv_url)
+    print(f'students_dataframe: \n{students_dataframe}')
+    students_file_path = f'lang-{language}/students.csv'
+    print(f'students_file_path: {students_file_path}')
+    students_dataframe.to_csv(students_file_path, index=False)
+
+    for index, student in students_dataframe.iterrows():
```

Ruff (0.11.9)

Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)